h2o-genmodel/src/main/java/hex/genmodel/algos/targetencoder/TargetEncoderMojoModel.java

```diff
@@ -14,12 +14,23 @@ public static double computeLambda(long nrows, double inflectionPoint, double sm
   public static double computeBlendedEncoding(double lambda, double posteriorMean, double priorMean) {
     return lambda * posteriorMean + (1 - lambda) * priorMean;
   }
+  
+  static Map<String, Integer> name2Idx(String[] columns) {
+    Map<String, Integer> nameToIdx = new HashMap<>(columns.length);
+    for (int i = 0; i < columns.length; i++) {
+      nameToIdx.put(columns[i], i);
+    }
+    return nameToIdx;
+  }
+
 
   public final Map<String, Integer> _columnNameToIdx;
   public Map<String, Boolean> _teColumn2HasNAs; // tells if a given encoded column has NAs
   public boolean _withBlending;
   public double _inflectionPoint;
   public double _smoothing;
+  public List<ColumnsToSingleMapping> _inencMapping;
+  public List<ColumnsMapping> _inoutMapping;
```

Comment:
@sebhrusen Is there any chance I could get this piece of information also for TE Model (not MOJO) via REST API. I need need it for multinomial cases. (Need to get the list of output columns for an input feature)

Reply:
sure, i can make it accessible in the model output

Reply:
@mn-mikke this is now exposed in the model output as `input_to_output_columns`

Reply:
Will try. Thanks a lot!